selectors/views/channel.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+export const makeGetChannelPostStatus = () => createSelector(
+    (state) => state.views.channel.channelPostsStatus,
+    (_, channel) => (channel.id),
```

Comment:
Maybe we just need to pass channelId, like `(_, channelId) => (channelId),`?  Same for below.

Reply:
Yes of course will this